src/LibraryManager/Configuration/ProxySettings.cs

```diff
@@ -45,7 +45,8 @@ private bool CheckForProxyBypass(Uri uri)
             if (_bypassUris == null)
             {
                 _bypassUris = new List<Uri>();
-                if (_settings.TryGetValue(Constants.HttpProxyBypass, out string bypassValue))
+                if (_settings.TryGetValue(Constants.HttpsProxyBypass, out string bypassValue)
+                    || _settings.TryGetValue(Constants.HttpProxyBypass, out bypassValue))
```

Comment:
This might be a dumb question, but is it possible for Constants.HttpsProxyBypass to be specified (such that it is picked up here over the Http version) but not Constants.HttpsProxy (such that the Http version is picked up in TryConfigureProxy? It looks like the settings are json, so that might not be possible, but I wanted to confirm.

Reply:
In both the bypass and proxy configuration code paths, we prefer https over http.  However in the bypass case they can be mixed - for example, if you define only `http_proxy` and `https_proxy.bypass`, then you'll use the https bypass value.  However, I went with the assumption that someone will use either `http_proxy` or `https_proxy` variables and not both.  Since this code path doesn't know whether the proxy settings are using `http_proxy` or `https_proxy`, I'm inclined to keep the simpler code for now until we get feedback to do otherwise.

Reply:
Fair enough. Just wanted to make sure that was intentional.